apps/omg/lib/omg/state/core/metrics.ex

```diff
@@ -22,8 +22,8 @@ defmodule OMG.State.Core.Metrics do
 
   def calculate(%Core{utxos: utxos}) do
     [
-      {"unique_users", unique_users(utxos)}
-      | Enum.map(balance(utxos), fn {currency, amount} -> {"balance_" <> Encoding.to_hex(currency), amount} end)
+      {:unique_users, unique_users(utxos)}
```

Comment:
this module should be deprecated now and a telemetry handler should be used for data manipulation

Reply:
IMHO `:telemetry.execute` should not send all state. 
```Telemetry is a dynamic dispatching library for metrics and instrumentations.``` from telemetry README.md
I can be wrong, but it seems like a bad idea for the telemetry handler to calculate the status metrics.

Reply:
What makes you think that? telemtry allows you to install whatever handlers you want, even specific ones like OMG.TelemetryStateHandler.

Reply:
encapsulation?

Reply:
I know what you mean, but I don't think we want to encapsulate business with gathering metrics. 
I think what we would want is to encapsulate metrics calculation with telemetry, and isolate business logic.